src/for-developers/wallet-faucet.md

```diff
@@ -0,0 +1,22 @@
+# Fuel Wallet
+
+You can deploy a contract without signing with a wallet by running the following command:
+
+`forc deploy --unsigned`
+
+You can initialize a wallet and an account for your wallet to sign transactions
```

Comment:
This sentence is kind of weirdly placed? Kind of just floating there without continuity.